src/Tec/Plugin.php

```diff
@@ -193,6 +203,9 @@ public function detect_tribe_plugins() {
 		if ( $dep->is_plugin_active( 'Tribe__Tickets__Main' ) ) {
 			$this->et_active = true;
 		}
+		if ( $dep->is_plugin_active( 'Tribe__Tickets_Plus__Main' ) ) {
+			$this->etp_active = true;
+		}
```

Comment:
These could be simplified:
```suggestion

		$this->etp_active = $dep->is_plugin_active( 'Tribe__Tickets_Plus__Main' );
```

Reply:
This is done in a later pull with all the checks.